src/System.Runtime.Loader/tests/AssemblyLoadContextTest.cs

```diff
@@ -15,12 +15,18 @@ public class AssemblyLoadContextTest
     {
         private const string TestAssembly = "System.Runtime.Loader.Test.Assembly";
 
+        [ActiveIssue(/* dotnet/coreclr */ 1187, PlatformID.Windows)] // dependency on coreclr behavior, waiting for new coreclr
```

Comment:
I'll need to submit that separately, will remove from this PR for now. I tested it locally and it works, the CI is just using an older mscorlib (probably from last successful build).